shoop/core/cache/impl.py

```diff
@@ -72,7 +71,8 @@ def bump_version(self, cache_key):
         :type cache_key: str
         """
         namespace = _get_cache_key_namespace(cache_key)
-        version = str(time.time())
+        current_version = self.get_version(cache_key) or 1
+        version = current_version + 1
```

Comment:
Maybe it's better to use int in this case.  Anyway.. the docstring of get_version() should be updated, because now it returns int rather than str.